DirectX11/HackerDXGI.cpp

```diff
@@ -1947,6 +2148,26 @@ STDMETHODIMP HackerDXGISwapChain::GetDesc(THIS_
 	HRESULT hr = mOrigSwapChain->GetDesc(pDesc);
 	if (hr == S_OK)
 	{
+		if (pDesc)
+		{
+			//TODO: not sure whether the upscaled resolution or game resolution should be returned
```

Comment:
Yeah, i think it is the best way. Steam and uplay overlays seems to work correctly